src/screens/crew6/relayScreen.cpp

```diff
@@ -222,16 +222,20 @@ void RelayScreen::onDraw(sf::RenderTarget& window)
         bool near_friendly = false;
         foreach(SpaceObject, obj, space_object_list)
         {
-            if ((!P<SpaceShip>(obj) && !P<SpaceStation>(obj))
+            P<ShipTemplateBasedObject> stb_obj = obj;
+
+            if (!stb_obj
                 || (!obj->isFriendly(my_spaceship) && obj != my_spaceship))
             {
                 P<ScanProbe> sp = obj;
+
                 if (!sp || sp->owner_id != my_spaceship->getMultiplayerId())
                 {
                     continue;
                 }
             }
-            if (obj->getPosition() - target->getPosition() < 5000.0f)
+
+            if (obj->getPosition() - target->getPosition() < stb_obj->getShortRangeRadarRange())
```

Comment:
isn't it possible for stb_obj to be nullptr here for a probe?

Reply:
Yep. Rewrote this check to align with GuiRadarView's